### ISSUE TITLE
A11y_VSCode_ViewProblems_ScreenReader: NVDA does not read the state of the tab Problems, Output, Debug Console, Terminal as selected or non selected.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51967]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.3.1
ScreenReader Version: NVDA2018.2

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Go to View menu and Select Problems.
3. Check the state of all the tabs(Problems,Output,Debug Console, Terminal).

**Actual:**
NVDA does not read the state of the tab Problems, Output, Debug Console, Terminal as (selected or non selected). Same Issue has also been observed with JAWS.

**Expected:**
NVDA should read each tab with its state either selected or non selected.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to know which tab is in active state and what action needs to be performed.

**MAS Reference**
MAS1.3.1- https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference: 
[A11y VSCode Problems NVDA TabState.pptx](https://github.com/Microsoft/vscode/files/2105151/A11y.VSCode.Problems.NVDA.TabState.pptx)

Does this issue occur when all extensions are disabled?: Yes
